### DISCUSSION TITLE
Workflow runs in forks even though it's limited to the main repo

### DISCUSSION THREAD

**Discussion by vadi2** [community/community#24993] [Actions]
<p>My <a href="https://github.com/Mudlet/Mudlet/blob/d357ab1908825dfd77cc36dfda506e98b1b9ac6d/.github/workflows/update-translations.yml" rel="noopener nofollow ugc">workflow</a> is limited to run in the main repository only:</p>
<pre><code class="lang-yaml">jobs:
  update-and-commit-text:
    runs-on: ubuntu-latest
    if: github.repository == 'Mudlet/Mudlet'
    steps:
</code></pre>
<p>And that worked for a long while - yet recently, it started running in forked repositories as well, for example: <a href="https://github.com/vadi2/Mudlet/pull/38" class="inline-onebox" rel="noopener nofollow ugc">Update text for translation in Crowdin by github-actions · Pull Request #38 · vadi2/Mudlet · GitHub</a>. Has anything changed in this regard? How can I make it run in the main repo only?</p>

**Comment by dherbst** [comment, marked as answer]
<p>I have used <code>github.repository_owner</code> for this.</p>
<p><code>if: ${{ github.repository_owner == 'Mudlet' }}</code></p>
